lib/teleterm/clusters/cluster_gateways.go

```diff
@@ -45,7 +47,13 @@ func (c *Cluster) CreateGateway(ctx context.Context, params CreateGatewayParams)
 		return nil, trace.Wrap(err)
 	}
 
-	if err := c.ReissueDBCerts(ctx, params.TargetUser, db); err != nil {
+	routeToDatabase := tlsca.RouteToDatabase{
+		ServiceName: db.GetName(),
+		Protocol:    db.GetProtocol(),
+		Username:    params.TargetUser,
+	}
```

Comment:
Hmm, I was not aware of this - I just tested this and my postgres db cert omits the schema name extension (1.3.9999.2.4).  However, if I try to connect it uses my --db-user as the --db-name. Not sure where that's happening

Reply:
In case of Postgres and Mongo databaseName is part part of Wire database protocol: For instance: 
https://github.com/gravitational/teleport/blob/1b0a083b9df37a9390eaaf57c3308e4b2a1783b4/lib/srv/db/postgres/engine.go#L177

The `--db-name` sets the value in connection string that is used my pgwire protocol and propagated in StartupMessage so event if the `RouteToDatabase.Database` is not set we are obtaining database name that was set by --db-name from the database wire protocol.